third_party/tonic/logging/dart_error.h

```diff
@@ -7,13 +7,61 @@
 
 #include "third_party/dart/runtime/include/dart_api.h"
 
+#include "tonic/dart_persistent_value.h"
+
 namespace tonic {
 
 namespace DartError {
+using UnhandledExceptionReporter =
+    std::function<void(const std::string&, const std::string&)>;
+
 extern const char kInvalidArgument[];
+extern DartPersistentValue on_error;
+extern UnhandledExceptionReporter log_unhandled_exception;
```

Comment:
docstring for public variables

Reply:
Why is `on_error` public?  Isn't someone supposed to access it from `SetErrorHandler`?

Reply:
Same for `log_unhandled_exception`.

Reply:
Meant to remove these - an earlier version of this patch used them.